src/olympia/devhub/views.py

```diff
@@ -1576,8 +1592,21 @@ def _submit_source(request, addon, version, submit_page, next_view):
             )
             VersionLog.objects.create(version_id=version.id, activity_log=activity_log)
             form.save()
+            log.info(
+                '_submit_source, form saved, addon.slug: %s, version.pk: %s',
+                addon.slug,
+                version.pk,
+            )
+            timer.log_interval('2.form saved')
 
-        return redirect(next_view, *redirect_args)
+        result = redirect(next_view, *redirect_args)
+        log.info(
+            '_submit_source, redirecting to next view, addon.slug: %s, version.pk: %s',
+            addon.slug,
+            version.pk,
+        )
+        timer.log_interval('3.redirecting to next view')
```

Comment:
2. and 3. are always going to be same time - `redirect` is a shortcut to just building a 301 response object so returns very quickly.

Reply:
So I should just remove this one, as it won't be at all useful?

Reply:
I wouldn't have thought it'd be useful but stranger things have happened